source/support.html.erb

```diff
@@ -10,6 +10,8 @@ description: Help and support connecting to GovWifi
 
     <div class="column-two-thirds">
       <h1 class="heading-large">GovWifi support</h1>
+
+      <h2 class="heading-medium">GovWifi support</h2>
```

Comment:
What's the benefit of having this heading twice in an H1 and H2?

Reply:
It's what the spreadsheet says. The point I think, is that the page is "GovWifi Support" and then there are two types of support available, support for "GovWifi" and support for "Network administrators'. 

![Screenshot 2019-04-29 at 10 22 10](https://user-images.githubusercontent.com/2160769/56887139-ad691980-6a68-11e9-840e-e32afc9fae48.png)